src/Icon/index.js

```diff
@@ -63,10 +66,11 @@ const Icon = ({
     sizeNumber = '24'
   }
 
+  style['fill'] = fill;
```

Comment:
nit: I should have caught this before, but more of just semantics, I recommend setting this to a constant `iconStyles` first. Like this:

```
const iconStyles = style['fill'] = fill;
```

Then style below can just be `style: iconStyles` below.